x/distribution/client/cli/query.go

```diff
@@ -32,22 +32,22 @@ func GetCmdQueryParams(queryRoute string, cdc *codec.Codec) *cobra.Command {
 	}
 }
 
-// GetCmdQueryOutstandingRewards implements the query outstanding rewards command.
-func GetCmdQueryOutstandingRewards(queryRoute string, cdc *codec.Codec) *cobra.Command {
+// GetCmdQueryValidatorOutstandingRewards implements the query validator outstanding rewards command.
+func GetCmdQueryValidatorOutstandingRewards(queryRoute string, cdc *codec.Codec) *cobra.Command {
 	return &cobra.Command{
-		Use:   "outstanding-rewards",
+		Use:   "validator-outstanding-rewards",
 		Args:  cobra.NoArgs,
-		Short: "Query distribution outstanding (un-withdrawn) rewards",
+		Short: "Query distribution outstanding (un-withdrawn) rewards for a validator",
```

Comment:
does this include self-delegation rewards AND commission? we should probably indicate that here

Reply:
OR does this refer to all the tokens in the validator (whether they belong to validator-operator or the delegators)

Reply:
It includes all rewards to the validator and all its delegations, including commission.

Reply:
clarified